vignettes/mlr3spatiotempcv.Rmd

```diff
@@ -63,55 +63,67 @@ Currently, the following resampling methods are implemented:
 
 To create a spatial task, use class `TaskClassifST` for classification and `TaskRegrST` for regression tasks.
 Please make sure that your dataset comes with coordinates since they are needed during the construction of the class (see the next code chunk).
-In addition, one can specify whether the coordinates should be used as features during modeling.
-<!-- coordinates are put into another object? Question is if is important for the user to know that xy-columns will be removed from the feature dataset if he/she already specified that coords should not be used as predictors.-->
```

Comment:
They live in another field, yes. 

I though the arg name `coords_as_features` is descriptive enough. Do we need to make adjustments?

Reply:
NO, that's fine, I think it is clear enough what is happening in the adjusted version.